packages/toolbox/src/chainlinkConversionPathTools.ts

```diff
@@ -56,37 +54,35 @@ class ChainlinkConversionPathTools {
 
     this.creationBlockNumber = chainlinkConversionPath.getCreationBlockNumber(this.network);
 
-    this.maxRange = options?.maxRange || 5000;
+    this.maxRange = options?.maxRange || 1000000;
```

Comment:
Testing with this value: RPCs can easily handle this amount of block for such contracts with a very low amount of transactions.